apps/browser/src/vault/popup/components/vault/view.component.ts

```diff
@@ -255,15 +298,32 @@ export class ViewComponent extends BaseViewComponent {
   }
 
   close() {
-    this.location.back();
+    if (this.senderTabId) {
+      BrowserApi.focusTab(this.senderTabId);
+    }
+
+    if (this.inPopout) {
+      window.close();
+    } else {
+      this.location.back();
+    }
```

Comment:
```suggestion
    if (this.inPopout) {
      window.close();
      return;
    }
    
    this.location.back();
```

Mega-nitpick, what are your thoughts on [leveraging early returns](https://methodpoet.com/return-early-or-if-statement/) instead of `else`?